Description: Fix compilation with more recent versions of FPC
 This also improves code-quality and fixes warnings.
Forwarded: yes
Author: Matthias Klumpp <mak@debian.org>

--- a/unitmain.pas
+++ b/unitmain.pas
@@ -169,7 +169,7 @@
     procedure AddFileAndDirectoryList(SL: TStringList; sublevels: Integer);
     procedure UpdateFileCount;
     procedure SortListView(Lv:TListView; Index:integer; Reverse: Boolean);
-    function AddTask(ASongItem: TSongItem; AMediaGainAction: TMediaGainAction; AVolume: Double): Integer;
+    function AddTask(ASongItem: TSongItem; AMediaGainAction: TMediaGainAction; AVolume: Double): Integer; overload;
     function FitsTaskType(ATask: TMediaGainTask; AMediaGainAction: TMediaGainAction; ASongItem: TSongItem): Boolean;
     { private declarations }
   public
